ACE/protocols/ace/RMCast/RMCast.mpc

```diff
@@ -1,13 +1,24 @@
 //  -*- MPC -*-
 project(RMCast) : acelib, ace_output, install, threads {
   avoids = ace_for_tao
-  sharedname   = ACE_RMCast
+  sharedname = ACE_RMCast
   dynamicflags += ACE_RMCAST_BUILD_DLL
 
   specific {
     install_dir = ace/RMCast
   }
 
+  // *TODO*: this is essentially a workaround to be removed ASAP. On modern
```

Comment:
We also have other compilers than msvc on windows, probably this has to be specific to the microsoft compiler